hyperforce/modules/ROOT/pages/deploy-mule-apps-hyperforce.adoc

```diff
@@ -1,6 +1,6 @@
 = Deploying Mule Apps to Hyperforce
 
-When deploying Mule Apps to a Salesforce Hyperforce (Hyperforce) instance, you deploy via CloudHub 2.0 to a private or shared space. Consider the following guidelines.
+When deploying Mule Apps to a Salesforce Hyperforce (Hyperforce) instance, you deploy on CloudHub 2.0 to a private or shared space. Consider the following guidelines.
```

Comment:
Example of "on": https://docs.mulesoft.com/cloudhub-2/#why-deploy-on-cloudhub-2-0

"via" suggests something a bit different